source/deployment/sso-google.rst

```diff
@@ -0,0 +1,30 @@
+Google Single Sign-On (E20)
+===========================
+
+Available in `Enterprise Edition E20 <https://mattermost.com/pricing-self-managed/>`__.
+
+.. include:: ../cloud/cloud-administration/common-sso-google.rst
+
+Configure Mattermost ``config.json`` for Google Apps SSO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of using the System Console, you can add the Google settings directly to the ``config.json`` file directly on your Mattermost server.
```

Comment:
```suggestion
Instead of using the System Console, you can add the Google settings directly to the ``config.json`` file on your Mattermost server.
```